### PR TITLE
Add Ubuntu/Debian install instructions

### PR DESCRIPTION
**Pull request by saschagrunert**
We now have packages for them in kubic so we can add it as installation instruction as well.

### CODE REVIEW

**Comment by TomSweeneyRedHat** [comment]
LGTM
But the README.md is becoming rather long.  I wonder if we should consider moving everything under "Obtaining skopeo" to an install.md and add a link to that to the README.md....

**Comment by rhatdan** [comment]
Yes lets consolidate that under install.md and not in the readme.

**Comment by saschagrunert** [comment]
> Yes lets consolidate that under install.md and not in the readme.

Done :+1:

**Comment by rhatdan** [comment]
LGTM
Thanks @saschagrunert

File: install.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+```bash
+$ git clone https://github.com/containers/skopeo $GOPATH/src/github.com/containers/skopeo
+$ cd $GOPATH/src/github.com/containers/skopeo && make binary-local
+```
+
+### Building in a container
+
+Building in a container is simpler, but more restrictive:
+
+- It requires the `docker` command and the ability to run Linux containers
```
Comment (TomSweeneyRedHat) [inline install.md]: not for this review, but we should change this to Podman at some point.

**Comment by TomSweeneyRedHat** [comment]
LGTM, a couple of questions that could be addressed later.

File: install.md (hunk 1 above)
Comment (rhatdan) [reply install.md]: Yes definitely.

Comment (saschagrunert) [reply install.md]: I can do that, just a sec

Comment (saschagrunert) [reply install.md]: Done

**Comment by TomSweeneyRedHat** [comment]
hmm, GitHub ate a comment.  I asked if we should bump the golang min from 1.9 to 1.12.12 due to the CVE issues ine earlier variants of go.  Your call.

**Comment by saschagrunert** [comment]
> hmm, GitHub ate a comment. I asked if we should bump the golang min from 1.9 to 1.12.12 due to the CVE issues ine earlier variants of go. Your call.

Fixed as well :+1:

**Comment by TomSweeneyRedHat** [comment]
LGTM
and thanks a bunch @saschagrunert for your usual attention to detail!

**Pull request merged**